### PR TITLE
Adding sphere

### LINKED ISSUE #981: Space Suggestion: Sphere, $S^2$, Two-dimensional sphere
## Space Suggestion

Let $X$ be $S^2 := \\{x \in \mathbb{R}^3 : \left \| x \right \|_2 = 1\\}$, where $\left\| x \right\|_2$ denotes the standard Euclidean norm of $x$.

$X$ is homeomorphic to:

- One-point compactification of [Euclidean Plane R2](https://topology.pi-base.org/spaces/S000176)
- [Suspension](https://en.wikipedia.org/wiki/Suspension_(topology)) of [Circle](https://topology.pi-base.org/spaces/S000170)
- Double cover of [Projective Plane](https://github.com/pi-base/data/issues/893#issue-2647576807) (<- Link to previous open Issue)

## Rationale

This space appears in almost every book about and adjacent to topology. E.g., Engelking example 2.3.9. Also mentioned a few times, e.g., https://github.com/pi-base/data/issues/851#issuecomment-2453138651.

## Relationship to other spaces and properties

Will give an example for the following presently empty searches:
 - [π-Base, Search for `~Has a group topology + topological n-manifold + ~empty`](https://topology.pi-base.org/spaces?q=%7EHas+a+group+topology+%2B+topological+n-manifold+%2B+%7Eempty)
 - [π-Base, Search for `compact + Simply connected + ~contractible + locally connected + ~empty`](https://topology.pi-base.org/spaces?q=compact+%2B+Simply+connected+%2B+%7Econtractible+%2B+locally+connected+%2B+%7Eempty)

Will be the 6th example of:
 - [π-Base, Search for `Topological $n$-manifold + connected`](https://topology.pi-base.org/spaces?q=Topological+%24n%24-manifold+%2B+connected)

### PR DESCRIPTION
**Pull request by david20000813**
Closes #981.

(An extra comment: I did not expect just asserting the few properties discussed in the issue is enough to imply every possible trait, except the impossible one Cardinality = $\aleph_1$. That was truly a surprise.)

### CODE REVIEW

**Comment by yhx-12243** [comment]
In fact, P3 ($T_2$) + P123 (Locally $n$-Euclidean) can be further simplified to a single P124 (Topological $n$-manifold) as https://github.com/pi-base/data/issues/981#issuecomment-2508780910 said. But anyway, it's equivalent and up to you to decide.

**Comment by david20000813** [comment]
@yhx-12243 I know. I was the one who wrote that comment. It's just slightly easier this way, as I just needed toncopy from the traits of the circle.

**Comment by prabau** [comment]
> In fact, P3 ( T 2 ) + P123 (Locally n -Euclidean) can be further simplified to a single P124 (Topological n -manifold) as [#981 (comment)](https://github.com/pi-base/data/issues/981#issuecomment-2508780910) said. But anyway, it's equivalent and up to you to decide.

Also note that it may not be easier to check things.  We would have to check separately that the space is T2 and second countable.  It easier to just check Locally n-Euclidean , and derive the rest from embeddable in euclidean space.

File: spaces/S000169/properties/P000123.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000169
+property: P000123
+value: true
+refs:
+- wikipedia: Manifold
+  name: Manifold on Wikipedia
+---
+
+See {{wikipedia:Manifold}}.
```
Comment (StevenClontz) [inline spaces/S000169/properties/P000123.md:10]:
```suggestion
Each point has a neighborhood homeomorphic to {S176}. See also {{wikipedia:Manifold}}.
```

**Review by StevenClontz — APPROVED** [review]
I have one suggestion to consider, but going ahead and approving.

**Comment by prabau** [comment]
For P200 (simnply connected), I think one also needs to mention the space is path connected, as Hatcher Prop 1.14 is only about the fundamental group.  (Otherwise it's kind of circular: Path connected is now derived from simply connected.)

**Comment by david20000813** [comment]
@prabau Oh, sure. I thought of Hatcher's statement as implicitly implying $S^2$ is path connected, as otherwise the fundamental group, without specifying a base point, is not even well-defined. But it certainly won't hurt to make that more explicit.

**Pull request merged**